### DISCUSSION TITLE
Feature request - Add 3 month calendar view and stopwatch widgets and/or custom app launcher

### DISCUSSION THREAD

**Discussion by bobo7777** [martinet101/ElevenClock#1275] [General]
As a TClock user I am missing the ability to have an integrated stopwatch and three month calendar view from my clock app. Would it be possible to add these features into Eleven Clock?

![image](https://github.com/marticliment/ElevenClock/assets/80723202/3ad2d636-2479-43fd-8049-c9e571e26a7d) 
![image](https://github.com/marticliment/ElevenClock/assets/80723202/05c97d20-5901-4198-9b23-b716b2753132)

If this is not feasible, could we instead get the ability to define multiple custom applications to run and simply add the option to run each one for single/double/middle mouse click action?

Example:

![image](https://github.com/marticliment/ElevenClock/assets/80723202/f60c1bfd-b473-4727-b253-2425059e4dbf)

Thanks

**Comment by marticliment** [comment]
I think it would be easier to run a custom command on click.

**Reply by marticliment** [reply]
I have migrated this to #1276
